frontend/workflows/projectCatalog/src/catalog/project-card.tsx

```diff
@@ -42,34 +40,31 @@ const ProjectCard = ({ project, onRemove }: ProjectCardProps) => {
 
   return (
     <StyledCard>
-      <Grid container justify="flex-end">
-        <Grid item className="showOnHover">
-          <IconButton size="small" variant="neutral" onClick={remove}>
-            <CloseIcon />
-          </IconButton>
-        </Grid>
-      </Grid>
       <Grid container wrap="nowrap">
-        <Grid container item direction="row" alignItems="flex-end">
-          <Grid item xs={1}>
-            <LanguageIcon language={(project?.languages || [])[0]} />
+        <Grid container item direction="row" alignItems="center">
+          <Grid item xs={10}>
+            <Typography variant="h6" color="secondary">
+              {project?.name?.toUpperCase()}
+            </Typography>
           </Grid>
-          <Grid item xs={11}>
-            <Typography variant="caption2">{project?.name}</Typography>
+          <Grid container item className="showOnHover" justify="flex-end" xs={2}>
+            <IconButton size="small" variant="neutral" onClick={remove}>
+              <CloseIcon color="secondary" />
+            </IconButton>
           </Grid>
         </Grid>
       </Grid>
       <Grid
         container
-        style={{ marginTop: "16px", paddingRight: "16px", flex: "1", overflow: "hidden" }}
+        style={{ marginTop: "8px", paddingRight: "16px", flex: "1", overflow: "hidden" }}
         zeroMinWidth
       >
-        <Typography variant="body2">{project.data?.description}</Typography>
+        <Typography variant="body2" color="rgba(13, 16, 48, 0.65)">
+          {project.data?.description}
+        </Typography>
       </Grid>
-      <Grid container justify="flex-end">
-        <Grid item className="showOnHover">
-          <KeyboardArrowRightIcon />
-        </Grid>
+      <Grid container item justify="flex-end" alignItems="center">
+        <KeyboardArrowRightIcon color="secondary" />
```

Comment:
Why are we showing this on all cards always?

Reply:
it conveys to the users that cards are clickable and go somewhere without having to hover over one.

Reply:
maybe we can make it a bit lighter. It seems kind of overwhelming if there are several projects
![Screenshot from 2022-04-06 11-36-56](https://user-images.githubusercontent.com/1004789/162045029-63fca52b-bcee-48e4-ae6c-42b66aa120ae.png)